test/archetype.test.ts

```diff
@@ -966,6 +966,8 @@ describe("Factory", function () {
     const nftMint = Archetype.attach(newCollectionAddressMint);
 
     await nftBurn.connect(owner).enableBurnToMint(nftMint.address, false, 2, 0, 5000);
+    // here
```

Comment:
Setting the burnConfig.limit to 5000 threw me off, because on line 227 of Archetype.sol, this condition returns false in the tests, which won't update the mapping.

This should be corrected somehow, because if the user sets their burn limit equal to their max supply the _minted mapping won't be updated.

Reply:
ok I understand now, thanks.
If it's same or greater, then there's no good reason to keep track of it or call minted() at all, because the minter doesn't have to worry about hitting any limits.

Reply:
Yep exactly